### PR TITLE
RFE: check syntax and build with Travis

### PR DESCRIPTION
**Pull request by WOnder93**
Let's automate this...

Example green run: https://travis-ci.org/WOnder93/audit-testsuite/builds/457895458
Example red run: https://travis-ci.org/WOnder93/audit-testsuite/builds/457896839

Note that you will need to enable the project on the https://travis-ci.org/ website for this to take effect.

### CODE REVIEW

**Comment by WOnder93** [comment]
I have updated the PR to properly install `yapf` as well. There don't seem to be any Python files in the testsuite right now, but just for good measure...

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    packages:
+      - astyle
+      - perltidy
+      - python-pip
+
+install:
+  - sudo pip install yapf
+
+script:
+  - tools/check-syntax -f && git diff && [ `git diff | wc -l` -eq 0 ]
```
Comment (pcmoore) [inline .travis.yml]: Why not the following?
```
  - tools/check-syntax -f && git diff --exit-code
```

Comment (WOnder93) [reply .travis.yml]: Good point, I didn't realize that git diff has such option. Thanks!

**Comment by pcmoore** [comment]
Merged via afb6db835498b64a65c8a039ffd90000e3fdc08e and 0b7430f4cfd2975066bbca23068cc45f9fd3a3be, thanks.  However, I'm going to leave this PR open until I've got Travis setup for this project ... working on that now.

**Comment by pcmoore** [comment]
Should be working now, added a badge to the README.md via 3c77573f6f9e15c0edc920b45e54d973501b700d.

**Pull request closed without merging**